### PR TITLE
NewMobies

### PR DESCRIPTION
**Pull request by Boy0000**
…s lightnings PR adds these).

Adding mobsounds to several of the existing mobs
Fixed diamond_sword.json to add mobs and include all mobs in lightnings PR aswell (need to remove the diamond_sword.json from his PR though)

### CODE REVIEW

File: assets/minecraft/models/item/diamond_sword.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
 		{ "predicate": {"custom_model_data": 15}, "model": "item/mob/rohana/rohana_walking"},
 		{ "predicate": {"custom_model_data": 16}, "model": "item/mob/rohana/rohana_hit"},
 		{ "predicate": {"custom_model_data": 17}, "model": "item/mob/tamaugachi/tamaugachi"},
-		{ "predicate": {"custom_model_data": 18}, "model": "item/mob/tamaugachi/tamaugachi"},
+		{ "predicate": {"custom_model_data": 18}, "model": "item/mob/tamaugachi/tamaugachi_move"},
```
Comment (0ffz) [inline assets/minecraft/models/item/diamond_sword.json:24]: I don't see this in the rp

**Review by 0ffz — CHANGES_REQUESTED** [review]
How come some mob drop textures have been removed? Also the question about tamaugachi moving model. Other than that looks good to me

File: assets/minecraft/models/item/diamond_sword.json (hunk 1 above)
Comment (Boy0000) [reply assets/minecraft/models/item/diamond_sword.json:24]: its in lightnings PR, the one for hiso etc which you have "required changes for (should also have solved those).
The foodstuff is just removing old ones that were never used, duplicates or just wrongly named

Comment (0ffz) [reply assets/minecraft/models/item/diamond_sword.json:24]: Alright then

**Review by 0ffz — APPROVED** [review]

**Pull request merged**